examples/scripts/ct_astra_weighted_tv_admm.py

```diff
@@ -134,11 +134,17 @@ def postprocess(x):
 
 where
 
-  $$W = \mathrm{diag}\left\{\exp( \sqrt{\mathbf{y}}) \right\} \;.$$
+  $$W = \mathrm{diag}\left\{ \mathrm{counts} / I_0 \right\} \;.$$
+
+The data fidelity term in this formulation follows
+:cite:`sauer-1993-local` (9)
+except for the scaling by $I_0$,
+which we use to maintain balance between
+the data and regularization terms if $I_0$ changes.
```

Comment:
This unusual approach to newlines is fine in the main docs, which are primarily viewed in rendered form, but it's perhaps undesirable here since users may wish to work directly with the source example scripts.

Reply:
fixed